app/src/main/java/com/nervousfish/nervousfish/activities/BluetoothConnectionActivity.java

```diff
@@ -36,6 +36,8 @@
 
 /**
  * This Bluetooth activity class establishes and manages a bluetooth connection.
+ * It shows a screen with the Bluetooth devices with which the user is paired and other Bluetooth devices
```

Comment:
Class JavaDoc should make sense without the class itself, so could you start with "{@link Activity} that..."